actors/evm/src/lib.rs

```diff
@@ -96,13 +95,7 @@ impl EvmContractActor {
             return Err(ActorError::illegal_argument("no initcode provided".into()));
         }
 
-        // create an empty storage HAMT to pass it down for execution.
-        let mut hamt = Hamt::<_, U256, U256>::new(rt.store().clone());
-
-        // create an instance of the platform abstraction layer -- note: do we even need this?
-        let mut system = System::new(rt, &mut hamt).map_err(|e| {
-            ActorError::unspecified(format!("failed to create execution abstraction layer: {e:?}"))
-        })?;
+        let mut system = System::create(rt)?;
```

Comment:
@vyzo its just a way to get `System` before is made, though API could probably change to avoid needing this.